tests/test_resources.py

```diff
@@ -8,25 +8,26 @@
 def test_worker_resources_display(hq_env: HqEnv):
     hq_env.start_server()
     hq_env.start_worker(
-        cpus=4,
+        cpus="4x2",
         args=[
             "--resource",
             "potato=range(1-12)",
             "--resource",
             "fairy=sum(1000_1000)",
             "--resource",
-            "shark=list(1,3,5,2)",
+            "shark=[1,3,5,2]",
         ],
     )
     table = hq_env.command(["worker", "list"], as_table=True)
     assert table.get_column_value("Resources") == [
-        "1x4 cpus; fairy Sum(10001000); potato Range(1-12); shark List(1,2,3,5)"
+        "cpus 4x2; fairy 10001000; potato 12; shark 4"
     ]
 
     table = hq_env.command(["worker", "info", "1"], as_table=True)
+    print(table.get_row_value("Resources"))
     assert (
         table.get_row_value("Resources")
-        == "1x4 cpus\nfairy: Sum(10001000)\npotato: Range(1-12)\nshark: List(1,2,3,5)"
+        == "cpus: 4x2\nfairy: 10001000\npotato: 12\nshark: 4"
```

Comment:
Why isn't `Range`/`Sum` printed here anymore?

Reply:
It is because full description of lists is enumeration of indices; that is long for Karolina (128 ids). So I would leave it as is now as default and add "-v" for some verbose mode that prints it in detail. For a user it is mostly important how many resources it have, not particular ids.

Reply:
Shouldn't we normalize these things anyway? Turn 0-127 into Range(128)? But it would have to work recursively for NUMA groups (Group(Range(16))), which is probably too annoying.

Reply:
Maybe we can just do "ranging" only in the output, something like [[0-11], [12-24]] (it is actually implemented in task ids), but still I believe that for most user it is more important to see that it is "2x12", rather then a precise ID (at least as default view).

Reply:
Ok, in most cases it will be range(N) anyway, i.e. numbers from 0 to N - 1.